.github/workflows/test_cov.yaml

```diff
@@ -0,0 +1,46 @@
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+
+---
+name: Test coverage
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.6
+          bundler-cache: true
+
+      - name: Cache python
+        uses: actions/cache@v2
+        with:
+          path: /opt/hostedtoolcache/Python/3.8
+          key: test-cov-ubuntu-18.04-3.8-${{ hashFiles('requirements.txt') }}
+
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Install dependencies
+        run: |
+          pip install -r requirements.txt
+          pip install coverage
+          pip install coveralls==2.2
+
+      - name: Test with pytest
+        run: pytest tensorbay --cov
```

Comment:
Will this new workflow run redundant unit-test?
Is it possible to add coverage test to `unit test` workflow?